### DISCUSSION TITLE
Motif Finding Algorithms/VF2++ subgraph isomorphism

### DISCUSSION THREAD

**Discussion by dschult** [networkx/networkx#4453] [Algorithms]
Finding how many subgraphs have a specified motif is a common analysis performed in e.g. gene regulatory networks. I think we have this in our MultiGraph Monomorphism finding code. 

How could we make this more accessible to motif finders? What features should we provide?

**Comment by aanastasiou** [comment]
[I am interested](https://github.com/aanastasiou/pymotifcounter) :) Can you elaborate a bit more?
